back/spec/services/export/custom_field_for_export_spec.rb

```diff
@@ -74,6 +74,53 @@
           expect(report_field.value_from(model)).to eq ''
         end
       end
+
+      context 'domicile fields' do
+        let(:custom_field) do
+          create(
+            :custom_field,
+            code: 'domicile',
+            key: 'domicile',
+            input_type: 'select',
+            title_multiloc: {
+              'en' => 'Residence',
+              'nl-NL' => 'Verblijfplaats'
+            }
+          )
+        end
+
+        context 'when there are no areas' do
+          let(:model) { create(:user) }
+
+          it 'returns the empty string' do
+            expect(report_field.value_from(model)).to eq ''
+          end
+        end
+
+        context 'when the field value is "outside"' do
+          let(:model) { create(:user, custom_field_values: { 'domicile' => 'outside' }) }
+
+          it 'returns empty string' do
+            expect(report_field.value_from(model)).to eq 'Somewhere else'
```

Comment:
In the original test that this was moved from, this was `eq ''` but this I think was incorrect as the value that a user will have selected will be 'Somewhere else'